modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -147,8 +147,8 @@ RETURN p
 ----
 
 Description of the returned code::
-Using shortest path with an exhaustive search fallback might cause query slow down since shortest path graph algorithms might not work for this use case.
-It may be possible to introduce a `WITH` to separate the `MATCH` containing the shortest path from the existential predicates on that path.
+Using shortest path with an exhaustive search fallback might result in a slower query since shortest path graph algorithms might not work for this use case.
+It may be possible to introduce a `WITH` to separate the `MATCH` containing the shortest path from the existential predicates on that path and thereby eliminating the need for the fallback.
```

Comment:
@renetapopova I can't remember, is this supposed to be the exact old 5.x description? In that case we probably should not have changed it...

Reply:
Yes, this is the currently returned description. So, changing it, I guess, would be a breaking change.

Reply:
I changed it back now